### PR TITLE
[React] Display API Error message on Login and Register failure

### LINKED ISSUE #530: [React] Web App doesn't show error message on Login/Register failure
**Describe the bug**
Currently, the Web App doesn't show any error message when login/register fails.

**To Reproduce**
Trying to log in with the wrong email or password
Trying to use already registered email or username for registration

**Expected behavior**
Should show the appropriate error message fetched from API.

**Screenshots**
label

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by amlannandy**
# Description

Earlier, the web app didn't show any message on login/register failure. Now it shows the Error message received from API.

Fixes #530 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

1. Trying to login with wrong email or password
2. Trying to use already used email or username for registration

**Test Configuration**:

Google Chrome, Windows 10

label2

label3

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by amlannandy** [comment]
@UdeshUK Hello. Could you please review this?

**Comment by carpecodeum** [comment]
Use Username instead of UserName

File: labellab-client/src/actions/auth.js [hunk 1]
```diff
@@ -235,11 +238,12 @@ export const GithubOauth = (credentials, callback) => {
       headers: {
         'content-type': 'application/json',
         'Access-Control-Allow-Origin': 'http://localhost:3000',
-        'Accept': 'application/json'
```
Comment (carpecodeum) [inline labellab-client/src/actions/auth.js:238]: please include `accept` in inverted commas

Comment (amlannandy) [reply labellab-client/src/actions/auth.js:238]: My code formatter must have changed them. I'll change it back.

**Comment by amlannandy** [comment]
> Use Username instead of UserName

@carpecodeum Okay. Actually I'm just displaying the message that's been sent by the API. So, I'll change the original message from the API itself. 👍🏻

**Comment by amlannandy** [comment]
@carpecodeum Made the requested changes 👍🏻

**Comment by carpecodeum** [comment]
@amlannandy The tests are failing could you see where the issue is?

**Comment by amlannandy** [comment]
> @amlannandy The tests are failing could you see where the issue is?

No. I wasn't able to find the issue. I tested everything again and it seemed to work. The first build did pass and I don't see anything in the 2nd commit that could've made it fail.

**Comment by amlannandy** [comment]
@carpecodeum The build passed this time without any changes 🤘🏻

**Review by carpecodeum — APPROVED** [review]

**Pull request merged**